lemmatized_text/views.py

```diff
@@ -81,6 +82,24 @@ def text(request, pk):
     return render(request, "lemmatized_text/text.html", {"text": text})
 
 
+def edit(request, pk):
+    lemmatized_text = get_object_or_404(models.LemmatizedText, pk=pk, created_by=request.user)
+
+    if request.method == "POST":
+        form = LemmatizedTextEditForm(request.POST)
+        if form.is_valid():
+            # pass to a "lemmatized text method" to handle changes
+            lemmatized_text.handle_edited_data(request.POST.get("title"), request.POST.get("text"))
```

Comment:
This could take a while, depending on how much of the text needs to be re-lemmatized, so we may want to turn `handle_edited_data` into an async job as a follow-up improvement.

Reply:
Agreed!